TROUBLESHOOTING.md

```diff
@@ -193,7 +193,20 @@ only be enabled for debugging.
   moving to the next step.
 
 * ```XLA_USE_BF16```: If set to 1, tranforms all the _PyTorch_ _Float_ values into _BiFloat16_
-  when sending to the _TPU_ device.
+  when sending to the _TPU_ device. Note that when using `XLA_USE_BF16=1` tensor arithmetic will
+  be done in reduced precision and so tensors such as loss values and other metrics will not be
```

Comment:
I don't like mentioning "loss values" here. This gives the impressions that trained models aren't useful, and that's just not true.

Reply:
done